eclair-core/src/main/scala/fr/acinq/eclair/wire/Onion.scala

```diff
@@ -290,6 +295,7 @@ object Onion {
     FinalTlvPayload(TlvStream(AmountToForward(amount), OutgoingCltv(expiry), PaymentData(paymentSecret, totalAmount), TrampolineOnion(trampolinePacket)))
   }
 
+  val keySendTagValue = UInt64(5482373484L)
```

Comment:
Remove, just directly put this value inline in the `typecase` in the `onionTlvCodec` like we do for other records

Reply:
Done